reverse.js

```diff
@@ -34,6 +34,7 @@
         instruction,
         op,
         lop,
+        preop,
```

Comment:
let's call this `prevop` so that it's clearer what its role is.

Reply:
Ok, done